src/app/ReadHandler.h

```diff
@@ -323,15 +329,15 @@ class ReadHandler : public Messaging::ExchangeDelegate
     void GetSubscriptionId(SubscriptionId & aSubscriptionId) const { aSubscriptionId = mSubscriptionId; }
     AttributePathExpandIterator * GetAttributePathExpandIterator() { return &mAttributePathExpandIterator; }
 
-    /**
-     * Notify the read handler that a set of attribute paths has been marked dirty.
-     */
-    void SetDirty(const AttributePathParams & aAttributeChanged);
+    /// @brief Update mDirtyGeneration to the latest DirtySetGeneration of the InteractionModelEngine. As this might make the
```

Comment:
So... in terms of _interface_ what's happening here is that the ReadHandler is being notified that the path being passed in is now considered dirty.  Maybe it should be called `AttributePathIsDirty`?

The details of how a ReadHandler reacts to that seem like implementation details that should be documented in the implementation, not here.

The one thing that should be documented here, in addition to what was here before, is that this will schedule a reporting run if the handler becomes reportable as a result of this path being dirty.